### ISSUE TITLE
bicaption 报错：未定义的bi-first选项

### ISSUE THREAD

**Issue by sikouhjw** [nanmu42/CQUThesis#52]
问题：《[cquthesis.cls样式文件编译出错(line 339,340)](https://github.com/nanmu42/CQUThesis/issues/51)》
解决方案：《[xelatex 2019.12.27 以及更新版，bicaption 报错：未定义的 bi-first 选项 ](https://github.com/mohuangrui/ucasthesis/issues/223)》
建议修复

**Comment by nanmu42** [comment]
谢谢反馈，与 #51 相关。

我这边目前暂时没有时间做修复，欢迎提PR.

**Comment by EigenSolver** [comment]
bicaption在texlive2020之后无法预定义。
[Bicaption for longtables doesn't work
](https://tex.stackexchange.com/questions/283980/bicaption-for-longtables-doesnt-work)

> Update (2020-01-03)
> 
> Starting with v3.4h of the caption package options cannot be used anymore prior to definition. Therefore \captionsetup[bi-first]{bi-first} and \captionsetup[bi-second]{bi-second} has to be done after \DeclareCaptionOption{bi-first}..., otherwise the document will not compile anymore.

**Comment by EigenSolver** [comment]
修复很简单,cquthesis.dtx中
```
\captionsetup[bi-first]{bi-first}
\captionsetup[bi-second]{bi-second}
```
放在`\DeclareCaptionOption{bi-first}[]`和`\DeclareCaptionOption{bi-second}[]`之后就好

**Comment by nanmu42** [comment]
Resolved by #55.

感谢 @Neuromancer43 .
